descriptor.md

```diff
@@ -137,11 +140,17 @@ If a useful algorithm is not included in the above table, it SHOULD be submitted
 [SHA-256][rfc4634-s4.1] is a collision-resistant hash function, chosen for ubiquity, reasonable size and secure characteristics.
 Implementations MUST implement SHA-256 digest verification for use in descriptors.
 
+When the _algorithm identifier_ is `sha256`, the _encoded_ portion MUST match `/[a-f0-9]{64}/`.
+Note that `[A-F]` MUST NOT be used here.
```

Comment:
I think hex-encoding is specific to algorithm.
Depending on the algorithm, hex-encoded digest can be `somealgorithm+uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6`